src/ripple/protocol/impl/STVar.h

```diff
@@ -54,12 +57,12 @@ class STVar
 
     STVar (STBase&& t)
     {
-        p_ = t.move(sizeof(d_), &d_);
+        p_ = t.move(max_size, &d_);
```

Comment:
It's not a pessimization in the sense that 72 is the largest fixed size object we allow in an `STVar`. The next larger one is 104 bytes. Perhaps a comment to that effect is in order.

Yes, `sizeof(d_)` gives us 80, but I tend to think of the extra 8 bytes as "padding". Maybe accessing them is OK (in fact, it _probably_ is) but I still think it's better to only use what we asked for.